opendnp3/DNP3/DNPFromStream.h

```diff
@@ -21,7 +21,7 @@
 
```

Comment:
Recommend we verify if any of the changes in this file are required to fix the exception issue. If they are not needed we should remove them.